cmd/snap/cmd_list.go

```diff
@@ -124,6 +131,60 @@ func listSnaps(names []string, all bool) error {
 	w := tabWriter()
 	defer w.Flush()
 
+	switch format {
+	case "":
+		return outputSnapsDefault(w, snaps)
+	default:
+		return outputSnapsWithFormat(Stdout, snaps, format)
```

Comment:
could you pass `w` instead of `Stdout`, so things still align if the caller bothers to use tabs?